diff --git a/cpanspec b/cpanspec
index 58fdce5..d6f0fe4 100755
--- a/cpanspec
+++ b/cpanspec
@@ -1523,16 +1523,16 @@ END
 END
 
     $spec->close();
-    my ($fh, $filename) = File::Temp::tempfile;
-    if (-x "/usr/lib/obs/service/format_spec_file.files/prepare_spec") {
-        if (!system("/usr/lib/obs/service/format_spec_file.files/prepare_spec '$specfile' > '$filename'")) {
+#    my ($fh, $filename) = File::Temp::tempfile;
+#    if (-x "/usr/lib/obs/service/format_spec_file.files/prepare_spec") {
+#        if (!system("/usr/lib/obs/service/format_spec_file.files/prepare_spec '$specfile' > '$filename'")) {
             # don't want to reimplement cross-device rename
-            system("mv '$filename' '$specfile'");
-        }
-    }
-    else {
-        print STDERR "please install obs-service-format_spec_file\n";
-    }
+#            system("mv '$filename' '$specfile'");
+#        }
+#    }
+#    else {
+#        print STDERR "please install obs-service-format_spec_file\n";
+#    }
 
     build_rpm($specfile) if ($buildsrpm or $buildrpm);
 
